StortSpelProjekt/Game/src/Gamefiles/Components/PlayerInputComponent.cpp

```diff
@@ -256,12 +256,12 @@ void component::PlayerInputComponent::move(MovementInput* evnt)
 			if (vel == double3({ 0.0, 0.0, 0.0 }))
 			{
 				forward.normalize();
-				vel = { forward.x * BASE_VEL * DASH_MOD, forward.y, forward.z * BASE_VEL * DASH_MOD };
+				vel = { forward.x * m_MovementSpeed * DASH_MOD, forward.y, forward.z * m_MovementSpeed * DASH_MOD };
```

Comment:
plz tone down the dash a bit, you can now glitch through the thick outer walls and into the ground in the "inside area"